### PR TITLE
Custom On Screen Controller Layout

### PR DESCRIPTION
**Pull request by lhc70000**
**Description:**

- Three OSC layouts
- Don't hide UI if mouse locates in OSC
- Fix no fading animation for OSC

### CODE REVIEW

**Comment by saagarjha** [comment]
How are the OSC options different from each other?

**Comment by alejx** [comment]
@saagarjha 
a0d03d0a83824ed6f457b199d3a0fb5c

**Comment by saagarjha** [comment]
Ahh, got it–looks like I needed to restart IINA for the setting to take effect.

File: iina/MainWindowController.swift [hunk 1]
```diff
@@ -210,7 +214,11 @@ class MainWindowController: NSWindowController, NSWindowDelegate {
       return window?.standardWindowButton(.documentIconButton)?.superview?.subviews.flatMap({ $0 as? NSTextField }).first
     }
   }
-  @IBOutlet weak var controlBar: ControlBarView!
+
+  var currentControlBar: NSView?
+
+  @IBOutlet weak var controlBarFloating: ControlBarView!
+  @IBOutlet weak var controlBarBottom: NSVisualEffectView!
```
Comment (alejx) [inline iina/MainWindowController.swift]: Maybe `controlBarTied` is better?

**Comment by lhc70000** [comment]
Finished.

**Pull request merged**